resources/views/community/components/cards/user.blade.php

```diff
@@ -28,9 +29,13 @@
 }
 
 if (empty($userData)) {
-    $userData = Cache::store('array')->rememberForever('user:' . $userName . ':card-data', function () use ($userName) {
-        return User::firstWhere('User', $userName);
-    });
+    $userData = Cache::store('array')->remember(
+        $userCardDataCacheKey,
```

Comment:
Why pass this in instead of just calculating it here?

Reply:
My original implementation did precisely that, however deptrac throws an error if this Blade component attempts to directly use any class from `Support`. Slightly related: https://github.com/RetroAchievements/RAWeb/pull/1659#discussion_r1264519669